backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -29,19 +31,26 @@ const fetchReasonDescription = (context, assignmentReasonCode, caseNotesApi) =>
     .then(cellMoveReasonValues => cellMoveReasonValues.find(record => record.value === assignmentReasonCode))
     .then(cellMoveReasonValue => cellMoveReasonValue.text)
 
-const fetchWhatHappened = (
+const fetchWhatHappened = async (
   context,
   offenderNo,
   bookingId,
   bedAssignmentHistorySequence,
   caseNotesApi,
   whereaboutsApi
-) =>
-  whereaboutsApi
-    .getCellMoveReason(context, bookingId, bedAssignmentHistorySequence)
-    .then(cellMoveReason => caseNotesApi.getCaseNote(context, offenderNo, cellMoveReason.cellMoveReason.caseNoteId))
-    .then(caseNote => caseNote.text)
-
+) => {
+  try {
+    return await whereaboutsApi
+      .getCellMoveReason(context, bookingId, bedAssignmentHistorySequence)
+      .then(cellMoveReason => caseNotesApi.getCaseNote(context, offenderNo, cellMoveReason.cellMoveReason.caseNoteId))
+      .then(caseNote => caseNote.text)
+  } catch (err) {
+    if (err?.response?.status === 404) {
```

Comment:
We generally prefer this style of if block:
`
if (err?.response?.status === 404)  return 'Not entered'
`